### PR TITLE
Remove newline from export_vcf if no FORMAT fields

### PR DESCRIPTION
**Pull request by konradjk**

### CODE REVIEW

**Comment by konradjk** [comment]
Fixes #4048

File: python/test/hail/methods/test_impex.py [hunk 1]
```diff
@@ -166,6 +166,11 @@ def test_export_vcf(self):
         dataset_imported = hl.import_vcf('/tmp/sample.vcf')
         self.assertTrue(dataset._same(dataset_imported))
 
+        no_sample_dataset = dataset.filter_cols(False)
```
Comment (tpoterba) [inline python/test/hail/methods/test_impex.py]: I think you also need to select_entries to make this a good test.

Also what happens if you have no entry fields but you do have samples? 🤔

Comment (konradjk) [reply python/test/hail/methods/test_impex.py]: I'll add select_entries(), but I think the behavior if samples and no entry fields is beyond the scope of this PR?

Comment (tpoterba) [reply python/test/hail/methods/test_impex.py]: yeah, totally. I'm just wondering.

**Pull request merged**